fm.reaper.Reaper.metainfo.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>fm.reaper.Reaper</id>
+  <name>Reaper</name>
+  <summary>Digital Audio Workstation.</summary>
+  <url type="homepage">https://www.reaper.fm/</url>
+  <url type="bugtracker">https://forum.cockos.com/forumdisplay.php?f=22</url>
+  <url type="help">https://forum.cockos.com/</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.reaper.fm/dist-agreement.php</project_license>
+  <description>
+    <p>REAPER is a complete digital audio production application for computers, offering a full multitrack audio and MIDI recording, editing, processing, mixing and mastering toolset.  </p>
+    <p> Using REAPER you can import any audio and MIDI, synthesize, sample, compose, arrange, edit, mix, and master songs or any other audio projects. If you add a hardware audio interface and a microphone, you have a complete professional quality recording studio, suitable for recording anything from a soloist to a band to an orchestra.</p>
+    <p>This is an unofficial version.</p>
+  </description>
+  <content_rating type="oars-1.0">
+    <content_attribute id="money-purchasing">mild</content_attribute>
+  </content_rating>
+
+  <launchable type="desktop-id">fm.reaper.Reaper.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Overview</caption>
+      <image>https://www.reaper.fm/v6img/ss_persp_v63.jpg</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="v6.61" date="2022-06-17">
+      <description>
+        <span>ReaSamploMatic5000</span>
```

Comment:
According to the [Appstream specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description) `<span>` is not allowed. appstreamcli, which is the more modern reference implementation shows the validation error:
```
$ appstreamcli validate fm.reaper.Reaper.metainfo.xml
E: fm.reaper.Reaper:229: description-markup-invalid span
E: fm.reaper.Reaper:80: description-markup-invalid span
E: fm.reaper.Reaper:95: description-markup-invalid span
E: fm.reaper.Reaper:43: description-markup-invalid span
E: fm.reaper.Reaper:193: description-markup-invalid span
I: fm.reaper.Reaper:5: summary-has-dot-suffix Digital Audio Workstation.
E: fm.reaper.Reaper:223: description-markup-invalid span
E: fm.reaper.Reaper:105: description-markup-invalid span
E: fm.reaper.Reaper:30: description-markup-invalid span
E: fm.reaper.Reaper:187: description-markup-invalid span
E: fm.reaper.Reaper:120: description-markup-invalid span
E: fm.reaper.Reaper:75: description-markup-invalid span
E: fm.reaper.Reaper:150: description-markup-invalid span
E: fm.reaper.Reaper:217: description-markup-invalid span
E: fm.reaper.Reaper:90: description-markup-invalid span
E: fm.reaper.Reaper:181: description-markup-invalid span
E: fm.reaper.Reaper:115: description-markup-invalid span
E: fm.reaper.Reaper:100: description-markup-invalid span
E: fm.reaper.Reaper:211: description-markup-invalid span
E: fm.reaper.Reaper:130: description-markup-invalid span
E: fm.reaper.Reaper:175: description-markup-invalid span
E: fm.reaper.Reaper:138: description-markup-invalid span
E: fm.reaper.Reaper:70: description-markup-invalid span
E: fm.reaper.Reaper:85: description-markup-invalid span
E: fm.reaper.Reaper:63: description-markup-invalid span
E: fm.reaper.Reaper:205: description-markup-invalid span
E: fm.reaper.Reaper:56: description-markup-invalid span
E: fm.reaper.Reaper:49: description-markup-invalid span
E: fm.reaper.Reaper:235: description-markup-invalid span
E: fm.reaper.Reaper:161: description-markup-invalid span
E: fm.reaper.Reaper:169: description-markup-invalid span
E: fm.reaper.Reaper:199: description-markup-invalid span
E: fm.reaper.Reaper:110: description-markup-invalid span
E: fm.reaper.Reaper:125: description-markup-invalid span

Validation failed: errors: 33, infos: 1
```